packages/app/package.json

```diff
@@ -211,10 +212,10 @@
     "lru-cache": "^4.1.3",
     "match-sorter": "^1.8.1",
     "memoize-one": "^4.0.0",
-    "mobx": "^4.0.0",
-    "mobx-react": "^5.2.3",
-    "mobx-react-lite": "^1.3.2",
-    "mobx-state-tree": "^3.3.0",
+    "mobx": "^5.11.0",
```

Comment:
Is there anything breaking in these major version upgrades? I'm quite scared if Cerebral likes this since it uses mobx extensively.

Reply:
https://github.com/mobxjs/mobx#mobx-4-vs-mobx-5

Doesn't appear that there are any breaking changes besides dropped IE11 support

https://github.com/mobxjs/mobx-react#choosing-your-version

mobx-react 6 adds hooks support by integrating mobx-react-lite into the main package. AFAIK this means there's no big difference between the two, and the long-term goal will be to replace mobx-react with mobx-react-lite entirely. Updating this dependency to latest is just to keep up to date with the latest bug fixes.

A quick look around the app doesn't seem to reveal anything broken. If there is, it's likely an extreme edge case. We can always roll back these deps if necessary.

Reply:
@Saeris Dropping IE11 support is quite a major one, since we still support IE11. @lbogdan has done some extra work for IE11 in #2176 recently too.

Reply:
What he's doing with that PR is specific to the embed. AFAIK we don't actually support IE11 for the full app. @CompuIves can correct me if I'm wrong. We already had a discussion on this.